com.endlessm.CompanionAppService.json.in

```diff
@@ -24,6 +24,27 @@
     ],
     "modules": [
         "com.endlessm.CompanionAppService.PipDependencies.json",
+        {
+            "name": "mustache-c",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/x86-64/mustache-c"
+                }
+            ]
+        },
+        {
+            "name": "meos-knowledge-content-renderer",
+            "buildsystem": "meson",
+            "builddir": true,
```

Comment:
I don't think `builddir` is necessary when using `meson` - it is enforced.

Reply:
Done.